lib/internal/util/parse_args/parse_args.js

```diff
@@ -229,7 +231,7 @@ function argsToTokens(args, options) {
       ArrayPrototypePush(
         tokens,
         { kind: 'option', name: longOption, rawName: arg,
-          index, value, inlineValue });
+          index, value, inlineValue, ...(helpValue !== undefined && { help: helpValue }) });
```

Comment:
I don't think the help text should be added to the token array. It doesn't help with interpretation of the parsing of the supplied arguments.

Reply:
Hey @shadowspawn thanks you for your time here!
You are right, I started the implementation for this part thinking that I would use the token value, I will make the necessary adjustments.